Assets/Scripts/Menu.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Menu : MonoBehaviour
+{
+    //Display Game Menu
+    void OnGUI()
+    {
+        const int width = 120, height = 60;
+
+        Rect buttonRect = new Rect(Screen.width / 2 - (width / 2), (2 * Screen.height / 3) - (height / 2), width, height);

```

Comment:
Dynamic sizing is present in a few places in the code you submitted, would there be a way to abstract the calculation somewhere so to increase the code's readability?